src/KeyringController.ts

```diff
@@ -167,15 +167,15 @@ class KeyringController extends EventEmitter {
   async setLocked(): Promise<KeyringControllerState> {
     delete this.password;
 
+    // remove keyrings
```

Comment:
Why was this change necessary? Just curious.

Reply:
It wasn't indeed, I believed `#clearKeyrings` looped over keyrings in the store, but they are an array of instances on the class property.
I rolled back the change